src/apps/content-editor/src/app/views/ItemEdit/components/Header/Header.js

```diff
@@ -1,47 +1,80 @@
-import { PureComponent } from "react";
+import React from "react";
 import cx from "classnames";
 
-import { PreviewUrl } from "./PreviewUrl";
-import { LiveUrl } from "./LiveUrl";
+import { actions } from "shell/store/ui";
+
 import { LanguageSelector } from "./LanguageSelector";
+import { useDispatch, useSelector } from "react-redux";
+import { ToggleButton } from "@zesty-io/core/ToggleButton";
 
 import ItemNavigation from "./ItemNavigation";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faDesktop, faWindowClose } from "@fortawesome/free-solid-svg-icons";
 
 import styles from "./Header.less";
-export class Header extends PureComponent {
-  render() {
-    return (
-      <header className={styles.Header}>
-        <div className={cx(styles.Split)}>
-          <div className={styles.Left}>
-            <ItemNavigation
-              modelZUID={this.props.modelZUID}
-              itemZUID={this.props.itemZUID}
-              item={this.props.item}
-            />
-          </div>
-          <div className={styles.Right}>
-            <div className={styles.Links}>
-              {this.props.item.web.path && <LiveUrl item={this.props.item} />}
+export function Header(props) {
```

Comment:
switching from a `PureComponent` to a function component has meaningful impacts on rendering. Is there a reason this change was made?